build.sbt

```diff
@@ -261,6 +261,9 @@ lazy val docs = http4sProject("docs")
         examplesWar,
         loadTest
       ),
+    scalacOptions in Tut ~= {
+      _.filterNot(Set("-Ywarn-unused:params", "-Ywarn-unused:patvars")) :+ "-Xfatal-warnings"
```

Comment:
`-Ywarn-unused:params` is due to undefined functions like: `def deleteTweet(id: Int): IO[Unit] = ???`.

`-Ywarn-unused:patvars` is due to `case request @ _ ...`, don't know how we want to handle those.

Reply:
Is it because `request` isn't referenced?  We can drop the `request @` in that case.  It only needs to be captured if it's going to be used.

Reply:
yes, it is, I'll remove the unnecessary ones :+1: